extensions/notebook/src/jupyter/jupyterSessionManager.ts

```diff
@@ -350,6 +350,10 @@ export class JupyterSession implements nb.ISession {
 					allCode += `%set_env ${key}=${process.env[key]}${EOL}`;
 				}
 			}
+
+			// Add and register Kqlmagic to the python kernel
+			allCode += `%reload_ext Kqlmagic`;
```

Comment:
What happens if users decline to update python packages? Does this fail and prevent them from being able to start notebooks?

Reply:
i.e. a user has python already. we ship the April release, where Kqlmagic is one of these packages. they decline to update packages, so they wouldn't have Kqlmagic.

Reply:
It doesn't fail to start notebooks. Notebook will work fine even if user declines to install Kqlmagic.